Cargo install fails with "Could not find `__rt` in `quote`"

This looks like a neat tool. Thank you for implementing it!

When I ran `cargo install clparse`, it failed with the following error:

```
error[E0433]: failed to resolve: could not find `__rt` in `quote`
   --> /home/emk/.cargo/registry/src/github.com-1ecc6299db9ec823/err-derive-0.1.6/src/lib.rs:145:63
    |
145 | fn display_body(s: &synstructure::Structure) -> Option<quote::__rt::TokenStream> {
    |                                                               ^^^^ could not find `__rt` in `quote`
```

--- a/.cl/fix-err-derive.yml
+++ b/.cl/fix-err-derive.yml
@@ -1,0 +1,2 @@
+---
+- changed: Bump err-derive from 0.1.6 to 0.2.4

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -36,6 +36,6 @@ clap = { version = "2.33.0", features = ["yaml", "suggestions", "color"] }
 serde_yaml = "0.8.9"
 indexmap = "1.2.0"
 anyhow = "1.0.3"
-err-derive = "0.1.6"
+err-derive = "0.2.4"
 derive-getters = "0.1.0"
 textwrap = "0.11.0"

